app/services/user_subscriptions/create.rb

```diff
@@ -0,0 +1,72 @@
+module UserSubscriptions
+  class Create
+    attr_accessor :user, :source_type, :source_id, :subscriber_email, :response
```

Comment:
Do these really need to be accessors, i.e. should the be writable by consumers of this service?

Reply:
Nope - updated!